### ISSUE TITLE
Use messages consistent with stylelint's conventions

### ISSUE THREAD

**Issue by jeddy3** [hudochenkov/stylelint-property-groups-structure#1]
Plugin looks great! I've one minor request.

Would it be possible make the messages consistent with stylelint's conventions? 

At the moment it says:

`Expected an empty line between property "${first}" and property "${second}"`,

In stylelint there is no concept of _between_ (with the exception of [`comment-empty-line-before`'s `"between-comments"`](https://github.com/stylelint/stylelint/tree/master/src/rules/comment-empty-line-before#between-comments), but that is [a mistake we intend to correct](https://github.com/stylelint/stylelint/issues/1663)), there is only [_before_ and _after_](https://github.com/stylelint/stylelint/blob/master/docs/user-guide/about-rules.md#-empty-line-before-and--max-empty-lines). And _after_ is restricted to optional secondary options. Only _before_ is used in empty line rule names and messages e.g.

`declaration-empty-line-before`:

``` js
export const messages = ruleMessages(ruleName, {
  expected: "Expected empty line before declaration",
  rejected: "Unexpected empty line before declaration",
})
```

Would it be possible to change this plugin's messages to?:

``` js
const messages = stylelint.utils.ruleMessages(ruleName, {
    expected: (property) => `Expected an empty line before property "${property}"`,
    rejected: (property) => `Unexpected an empty line before property "${property}"`,
});
```

We've endeavoured to make all the messages consistent because we believe it creates a better user-experience. I think it'd be to the betterment of the ecosystem if this carried through to plugins too.

**Comment by hudochenkov** [comment]
Great idea! I'll changed it.

**Comment by hudochenkov** [comment]
Released `v0.1.1`.
